### PR TITLE
remote: fix logical error in function transfer

### PR DESCRIPTION
**Pull request by pingl2016**
Check the value of variable before using it.

Signed-off-by: Ping Li <pingl@redhat.com>

### CODE REVIEW

**Comment by pingl2016** [comment]
ID: 1351002

File: virttest/remote.py [hunk 1]
```diff
@@ -806,7 +806,7 @@ def transfer(*args, **kwargs):
         start_time = time.time()
         ret = func(*args, **kwargs)
         elapsed_time = time.time() - start_time
-        if "filesize" in kwargs:
+        if "filesize" in kwargs and kwargs["filesize"] is not None:
```
Comment (xutian) [inline virttest/remote.py]: if kwargs.get("fileszie"): looks more compact

Comment (pingl2016) [inline virttest/remote.py]: yes, you are right. thanks. have updated it. could you help to review it again?

**Comment by xutian** [comment]
fine, thanks @pingl2016 .

**Comment by xutian** [comment]
merged, thanks @pingl2016

**Pull request merged**